<?xml version="1.0"?>

<launch>
	<node pkg="inspection_realsense" type="findinitialpose_actionserver" name="findinitialpose" output="screen"/>
	<node pkg="inspection_realsense" type="moveforward_actionserver" name="moveforward" output="screen"/>
	<node pkg="inspection_realsense" type="movebackward_actionserver" name="movebackward" output="screen"/>
	<node pkg="inspection_realsense" type="moveleftstep_actionserver" name="moveleftstep" output="screen"/>
        <node pkg="inspection_realsense" type="checkNewScanBackward_actionserver" name="checkNewScanBackward" output="screen"/>
        <node pkg="inspection_realsense" type="repub_cmd_vel" name="repub_cmd_vel" output="screen"/>


</launch>
